components/avatar/avatar.css

```diff
@@ -0,0 +1,37 @@
+.avatar {
+  width: 6rem;
+  height: 6rem;
+  border-radius: 50%;
+  margin: 0.5rem;
+  object-fit: cover;
+}
+
+.text-avatar {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--color-lynx-gray);
+    font-size: 3rem;
+}
+
+.avatar-s {
+    width: 4rem;
+    height: 4rem;
+}
+
+.avatar-m {
+    width: 8rem;
+    height: 8rem;
+}
+
+.avatar-l {
+    width: 10rem;
+    height: 10rem;
+}
```

Comment:
Just a suggestion, for sizes mostly convention is preferred as sm, md, lg.

Reply:
sure, will make the changes accordingly